WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
I hate any solution including the phrase "...you just have to remember..."!

You could make some kind of checksum on the seed file. But you'd have to store that in the DB. I suppose there could be a 'Seed Checksum" country in the test data... But I don't know how you deal with the checksum itself being in the checksummed file. I suppose it would have to be in a separate class And enforced with a test wherever it's kept.

A bit clunky, and I've never done a file checksum, but it should work, no?